src/frontends/pytorch/src/transforms/tuple_unpack_replacer.cpp

```diff
@@ -29,16 +31,25 @@ PrimTupleUnpackReplacer::PrimTupleUnpackReplacer() {
         OutputVector outputs;
         auto input_node = tuple_unpack->get_input_node_shared_ptr(0);
         auto tuple_construct = cast_fw_node(input_node, "prim::TupleConstruct");
-        if (!tuple_construct) {
-            return false;
-        }
-        for (const auto& input : input_node->inputs()) {
-            const auto& out = input.get_source_output();
-            outputs.push_back(out);
-        }
-        replace_node(tuple_unpack, outputs);
+        if (cast_fw_node(input_node, "prim::TupleConstruct")) {
+            for (const auto& input : input_node->inputs()) {
+                const auto& out = input.get_source_output();
+                outputs.push_back(out);
+            }
+            replace_node(tuple_unpack, outputs);
```

Comment:
can it be a case that not all outputs from TupleUnpack is used or output size is less that `outputs` size?

Reply:
No, I will add such case to tests